zenoh_c_vendor/CMakeLists.txt

```diff
@@ -26,10 +26,13 @@ set(ZENOHC_CARGO_FLAGS "--no-default-features$<SEMICOLON>--features=zenoh/shared
 # - https://github.com/eclipse-zenoh/zenoh-c/pull/620 (fix ze_querying_subscriber_get API to query newly discovered publishers)
 ament_vendor(zenoh_c_vendor
   VCS_URL https://github.com/eclipse-zenoh/zenoh-c.git
-  VCS_VERSION 134dbfa06ca212def5fb51dd8e816734dfd8dff6
+  VCS_VERSION a57fda736d2f8866c06509f34b5bc73e372c6aaf
   CMAKE_ARGS
     "-DZENOHC_CARGO_FLAGS=${ZENOHC_CARGO_FLAGS}"
+    "-DZENOHC_BUILD_WITH_UNSTABLE_API=TRUE"
```

Comment:
What is the unstable API that we need from Zenoh 1.0?  I'm not against doing this, but I do want to make sure we keep track of this.

Reply:
I'm afraid that your suggestion doesn't work. As I said, we need the unstable API to declare a SHM provider even though we don't really use it properly. If we remove this flag, rmw_zenoh_cpp won't compile.